### PR TITLE
Enhanced docs linking

### PR DESCRIPTION
**Pull request by githubtrainer**
The README stresses that this repo is intended for forks, but GitHub newbies
may have no idea what these are about, so we link to the official docs!

We provide even more links to GitHub help for newbies: branch management, issues, milestones, labels, and wikis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ A great example repo you can fork to your heart’s content, that serves as a la
 
 ## Getting started
 
-The first thing you want to do with this repo is quite likely **fork it**.  Use the Fork button in the top-right section of the page to do this.
+The first thing you want to do with this repo is quite likely [**fork it**](https://help.github.com/articles/fork-a-repo/).  Use the Fork button in the top-right section of the page to do this.
 
-Once you’re on your fork, you can experiment however you like with it, play with branches, issues, milestones, labels, the wiki, and more.
+Once you’re on your fork, you can experiment however you like with it, play with [branches](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/), [issues](https://help.github.com/articles/about-issues/), [milestones](https://help.github.com/articles/associating-milestones-with-issues-and-pull-requests/), [labels](https://help.github.com/articles/applying-labels-to-issues-and-pull-requests/), the [wiki](https://help.github.com/articles/about-github-wikis/), and more.
```
Comment (tdd) [inline README.md]: I believe https://help.github.com/articles/viewing-branches-in-your-repository/ would be a better entry point for branch help.

**Comment by tdd** [comment]
Hey trainer!

Thanks for helping improve the README. Overall it looks great, there's just a little link change I'd like you to consider.

File: README.md (hunk 1 above)
Comment (githubtrainer) [inline README.md]: Hey Chris,

That's a valid point, let me do that.

-- 
Christophe Porteneuve
Founder & CTO, Delicious Insights
christophe@delicious-insights.com
+33 662 059 656

**Comment by githubtrainer** [comment]
Hey!

Thanks for the feedback.  Glad you like it!

-- 
Christophe Porteneuve
Founder & CTO, Delicious Insights
christophe@delicious-insights.com
+33 662 059 656

**Comment by githubtrainer** [comment]
Hey!

Awesome, man.

Le 28/08/15 11:55, Christophe Porteneuve a écrit :
> Hey trainer!
>
> Thanks for helping improve the README. Overall it looks great, there's
> just a little link change I'd like you to consider.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/githubtrainer/oreilly-github-demo/pull/1#issuecomment-135723576>.
>


-- 
Christophe Porteneuve
Founder & CTO, Delicious Insights
christophe@delicious-insights.com
+33 662 059 656

**Comment by tdd** [comment]
This is too good.

On 28/08/15 11:59, Christophe Porteneuve wrote:
> Hey!
>
> Awesome, man.
>
> Le 28/08/15 11:55, Christophe Porteneuve a écrit :
>  > Hey trainer!
>  >
>  > Thanks for helping improve the README. Overall it looks great, there's
>  > just a little link change I'd like you to consider.
>  >
>  > —
>  > Reply to this email directly or view it on GitHub
>  >
> <https://github.com/githubtrainer/oreilly-github-demo/pull/1#issuecomment-135723576>.
>  >
>
>
> --
> Christophe Porteneuve
> Founder & CTO, Delicious Insights
> christophe@delicious-insights.com
> +33 662 059 656
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/githubtrainer/oreilly-github-demo/pull/1#issuecomment-135724254>.
>


-- 
Christophe Porteneuve
tdd@tddsworld.com

**Pull request merged**